packages/frontend/src/features/terminal/components/PerformanceMonitor.vue

```
@@ -0,0 +1,112 @@
+
+/** 上下文状态对应的指示颜色 */
+const contextDotColor = computed(() => {
+  switch (props.metrics.contextState) {
+    case 'active':
+      return 'bg-green-400';
+    case 'lost':
+      return 'bg-yellow-400';
+    case 'unavailable':
+      return 'bg-red-400';
+  }
+});
```

ESLint

[error] Expected to return a value in computed function.

(vue/return-in-computed-property)